independent-publisher-connectors/AADInvitationManager/readme.md

```diff
@@ -0,0 +1,43 @@
+## Azure AD Identity and Access
+
+The Graph API is a powerful and very extensive REST API. Using this API, you can create and manage objects (e.g. users, groups etc.) in Azure Active Directory, create Microsoft Teams teams, and a lot more.
+
+This connector exposes the Identity and Access resources of the Graph API in the Microsoft Power Platform.
+
```

Comment:
@Laskewitz The publisher section is missing, You can find a template for readme.md [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/readme.md). Can you please add it and other sections if you have valid data.

Reply:
Fixed

Reply:
@Laskewitz The publisher section is missing which is a required one. Can you please let me know if you are not able to access the template I have shared in my previous message?